src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -2120,6 +2120,7 @@ internal enum ErrorCode
         ERR_FilePathCannotBeConvertedToUtf8 = 9069,
         ERR_ReadOnlyNotSuppAsParamModDidYouMeanIn = 9070,
         ERR_FileLocalDuplicateNameInNS = 9071,
+        ERR_ScopedNotBeforeType = 9072,
```

Comment:
>ERR_ScopedNotBeforeType [](http://example.com/codeflow?start=8&length=23)

nit: MisplacedScoped? `scoped` is also allowed before `ref`

Reply:
> MisplacedScoped? `scoped` is also allowed before `ref`

Ref type is a type and the message makes it clear, I think